backend/utils/sheet.js

```diff
@@ -266,49 +266,73 @@ const getInstitutes_ = async () => {
   return getTableData(institutesList, rawData);
 };
 
-const deleteFromSheet = async (rowIndex, table) => {
-  try {
-    const auth = new google.auth.GoogleAuth({
-      keyFile: SERVICE_ACCOUNT_SECRET_FILE,
-      scopes: "https://www.googleapis.com/auth/spreadsheets",
-    });
-    const authClientObject = await auth.getClient();
-    const googleSheetsInstance = google.sheets({
-      version: "v4",
-      auth: authClientObject,
-    });
+const sendmail = require("../mail");
+const fs = require('fs').promises;
 
-    const spreadsheet = await googleSheetsInstance.spreadsheets.get({
-      spreadsheetId: SPREADSHEET_ID,
-    });
-    const sheets = spreadsheet.data.sheets;
-    const sheet = sheets.find(
-      (s) => s.properties.title === tableSheetMap[table]
-    );
-    const sheetId = sheet.properties.sheetId;
-
-    const result = await googleSheetsInstance.spreadsheets.batchUpdate({
-      spreadsheetId: SPREADSHEET_ID,
-      resource: {
-        requests: [
-          {
-            deleteDimension: {
-              range: {
-                sheetId: sheetId,
-                dimension: "ROWS",
-                startIndex: rowIndex,
-                endIndex: rowIndex + 1,
+const deleteFromSheet = async (rowIndex, table) => {
+  // console.log(table);
+  const result = getUsers_();
+  result.then(value => {
+  const email = value.rows[rowIndex - 1]['User Email'];
+    (async () => {
+      try {
+        const path = require('path');
+        const jsonString = await fs.readFile(path.join(__dirname, '..', 'template_mail.json'), 'utf8');
+        const data = JSON.parse(jsonString);
+        sendmail(email, data.emails.revoke_access);
+      } catch (err) {
+        console.error('Error reading or parsing file:', err);
+        throw err;
```

Comment:
Log the error and return from here instead of rethrowing it as in the access_grant case

Reply:
Done